src/constructors/element.js

```diff
@@ -1,8 +1,10 @@
+// @flow
 import { createElement } from 'react'
+import ValidRuleSetChild from '../models/ValidRuleSetChild'
 
 import Root from '../models/Root'
 
-const element = (tagName, ...rules) => {
+const element = (tagName: string, ...rules: Array<typeof ValidRuleSetChild>): Function => {
```

Comment:
This should probably be `: createElement` instead of `: Function` I just realized